pkgs/applications/office/banana-accounting/default.nix

```diff
@@ -0,0 +1,57 @@
+{ autoPatchelfHook
+, cairo
+, config
+, e2fsprogs
+, fetchurl
+, gmp
+, gtk3
+, libGL
+, libX11
+, lib
+, stdenv
+}:
+
+stdenv.mkDerivation {
+  pname = "banana-accounting";
+  version = "10.0.12";
+
+  srcs = fetchurl {
+    url = "https://www.banana.ch/accounting/files/bananaplus/exe/bananaplus.tgz";
```

Comment:
Just for info:
I have seen the download page, and indeed it does not version the file.